src/app/viewing/responses-list/responses-list.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="response-list-card d-none d-sm-none d-md-none d-lg-block mt-4">
+  <div class="card p-3 border-0">
+    <div class="row user-info">
+      <div class="col-lg-3 col-md-4">
+        <a [routerLink]=""><div class="profile-pic"></div></a>
+      </div>
+      <div class="col-lg-9 col-md-8">
+        <label class="mt-2 username">User Name</label>
+      </div>
+    </div>
+    <div class="dropdown-divider"></div>
+    <ul class="nav flex-column ml-2">
+      <li class="nav-item mb-2 response-title">
+        My Responses
+      </li>
+    </ul>
+    <div id="author-list">
+      <ul class="nav flex-column ml-2">
+        <li class="nav-item mb-1" *ngFor="let response of userMsgData" (click)="showMessage(response.authorid);">
+          {{ response.author }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+<div class="col-12 d-block d-sm-block d-md-block d-lg-none mt-4">
+  <div class="row user-info shadow rounded-pill py-2 my-3">
+    <div class="col-md-1 col-sm-2 col-2">
+      <a href="#"><div class="profile-pic"></div></a>
+    </div>
+    <div class="col-md-9 col-sm-8 col-8">
+      <label class="mt-2 username">User Name</label>
+    </div>
+    <div class="col-md-2 col-sm-2 col-2">
+      <button class="navbar-toggler hamburger-menu float-right" type="button" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" aria-expanded="false" aria-label="Toggle navigation">
+        <img src="../../../assets/images/icons/hamburger_menu.svg">
```

Comment:
Missing 'alt' attribute
```suggestion
        <img src="../../../assets/images/icons/hamburger_menu.svg" alt="Hamburger Menu">
```